README.md

```diff
@@ -572,7 +572,12 @@ The first step is to update the appropriate `DOCKER_HOST` in `druid_setup/.env`.
     - Historical config is located in `druid_setup/cluster/environment/historical.env` (`druid_host`)
   - Also, ensure that [public key authentication](https://kb.iu.edu/d/aews) is enabled.
 
-Next, deploy the approriate Druid cluster:
+Next, create the following directories on the remote server:
+
+- /home/share
+- /data/output
```

Comment:
do we want to give exact commands to run here? is there anything to be aware of permissioning wise to allow the pipeline and docker to write to these folders?

Reply:
The only permission issues here would be if these directories are owned by root. These should be either owned by a non root user or allow everyone to read/write to it. A note can be added to the README if required.